library/src/main/java/com/mapzen/android/MapView.java

```diff
@@ -126,6 +126,28 @@ public void getMapAsync(@NonNull String key,
     }
 
     /**
+     * Load map asynchronously using given API key.
+     *
+     * @param callback listener to be invoked when map is initialized and ready to use.
+     * @param key vector tiles API key that should be used to load map tiles.
+     * @param sceneFile scene file that should be used
+     */
+    public void getMapAsync(@NonNull String key,
+            @NonNull String sceneFile, @NonNull OnMapReadyCallback callback) {
+        mapInitializer.init(this, key, sceneFile, callback);
+    }
+
+    /**
+     * Load map asynchronously using given API key.
+     *
+     * @param callback listener to be invoked when map is initialized and ready to use.
+     * @param sceneFile scene file that should be used
+     */
+    public void getMapAsync(@NonNull OnMapReadyCallback callback, @NonNull String sceneFile) {
```

Comment:
We cant though because we already have a method with the same signature for the keys. It seems like we have a couple options though- 1. make all the getMapAsync methods which take a scene file have the scene file as the last param, 2. change the name of the method where we pass a scene file to something like getCustomMapAsync. Do you have a preference? I think I have a slight preference for changing the method name